src/replica/replica_stub.cpp

```diff
@@ -1716,6 +1772,10 @@ void replica_stub::on_gc()
     uint64_t cold_backup_running_count = 0;
     uint64_t cold_backup_max_duration_time_ms = 0;
     uint64_t cold_backup_max_upload_file_size = 0;
+    uint64_t bulk_load_running_count = 0;
```

Comment:
It's not this PR's problem, but these statistics are under function 'on_gc' seems strange. Could you please refactor it by another PR?

Reply:
Okay, I will try to refactor it further.